client/state/invites/actions.js

```diff
@@ -101,10 +100,11 @@ export function deleteInvite( siteId, inviteId ) {
 const deleteInvitesFailureNotice = ( siteId, inviteIds ) => ( dispatch, getState ) => {
 	for ( const inviteId of inviteIds ) {
 		const invite = getInviteForSite( getState(), siteId, inviteId );
+		const invitee = invite.user.email || invite.user.login;
 		dispatch(
 			errorNotice(
 				translate( 'An error occurred while deleting the invite for %s.', {
-					args: truncate( invite.user.email || invite.user.login, { length: 20 } ),
+					args: invitee.length > 20 ? invitee.slice( 0, 20 ) + '…' : invitee,
```

Comment:
After checking the [existing translations](https://translate.wordpress.com/projects/wpcom/-all-translated/35822/) for `…` (`&hellip`), it should be safe to assume that ellipsis character will be used as-is across pretty much all locales.

However, it might be a good idea to wrap the `…` itself in a `translate()` call to ensure it can be localized in case it needs to.

Reply:
Fixed via 62a50a6357d363ee3f52ad0a7d864f010d0ff8ee